content/clause-interleaving.md

```diff
@@ -103,14 +103,13 @@ It provides the best of all worlds:
 
 ### Specification
 We amend the syntax of def parameter clauses as follows:
-
 ~~~
 DefDcl                 ::=  DefSig ‘:’ Type
 DefDef                 ::=  DefSig [‘:’ Type] ‘=’ Expr
-DefSig                 ::=  id [DefParamClauses] [DefImplicitClause]
-DefParamClauses        ::=  DefParamClauseChunk {DefParamClauseChunk}
-DefParamClauseChunk    ::=  [DefTypeParamClause] TermOrUsingParamClause {TermOrUsingParamClause}
-TermOrUsingParamClause ::=  DefTermParamClause
+DefSig                 ::=  id [DefParamClauses] [DefImplicitClause]    -- and two DefTypeParamClause cannot be adjacent
+DefParamClauses        ::=  DefParamClause { DefParamClause }
```

Comment:
I think it would be clearer if the comment was on this line rather than the line above, but I'm really nitpicking here 😄

Reply:
I see what you mean, I guess it mostly depends on how we want to use `DefParamClauses` in other places

I also assumed people would be looking for it  first at the `DefSig` level, so it might be faster if the information is already here

Ultimately, I do not have a strong opinion either way, so maybe the other reviewers can comment on this ?

Reply:
I agree with @chrisandrews-ms. If @Kordyjan or @soronpo have a different opinion, please let us know!